app/views/provider/admin/user/notification_preferences/_form.html.slim

```diff
@@ -7,11 +7,11 @@
 
       ol
 
-        = f.input :enabled_notifications, as: :check_boxes_with_hints, required: false,
```

Comment:
is `required` intentionally removed?

Reply:
Inputs are not required by default so I thought we could get rid of some redundancy.